doc/tech/approach.md

```diff
@@ -14,17 +14,17 @@
 > _**Editor's Note:** The purpose in the subsections is to provide initial high-level content related to networking across CNTT, so as to provide general direction and help coordinate current independent networking development activities, as well as to satisfy the networking MVP deliverables for the Baldy release. The following content addresses the overall CNTT networking solution, and it is anticipated this content will be expanded and be relocated where appropriate._
 
 The basic approach to producing the initial deliverables is to:
-A) Summarize the overall CNTT Networking and Fabric Strategy
-B) Document the initial list of Objectives and Requirements
-C) Document enough initial logistical details for contributors to create coherent content
+A) Summarize the overall CNTT Networking Scope
+B) Document the initial Strategy and Principles
+C) Document the basic terms
+D) Provide a base Networking Reference Model within the Cloud Infrastructure context
```

Comment:
Wasn't is so, that it is not in the scope to create a new reference model, but the results of the Networking WS should be added to the RM and the RA-s?